app/templates/login.hbs

```diff
@@ -1,19 +1,35 @@
-<div class="panel panel-primary">
-  <div class="panel-heading">Sign in</div>
-  <div class="panel-body">
-    <form role="form">
-      <div class="form-group">
-        <label for="username">Username</label>
-        <input class="form-control" id="username">
-      </div>
-
-      <div class="form-group">
-        <label for="password">Password</label>
-        <input type="password" class="form-control" id="password">
-      </div>
-    </form>
-  </div>
-  <div class="panel-footer">
-    <p>Or <a {{action "loginAsGuest"}}>continue as a guest</a></p>
-  </div>
-</div>
\ No newline at end of file
+<div class="col-md-6">
+  {{#paper-card}}
+    {{#paper-card-content}}
+      <h3>Log In</h3>
+      <form role="form" {{action "login" on="submit"}}>
+          {{paper-text classNames="login-input" type="email" label="Email" value=loginEmail}}
+          {{paper-text classNames="login-input" type="password" label="Password" value=loginPassword}}
+        {{#paper-button type="submit" classNames="md-primary" raised=true}}
+          Log In
+        {{/paper-button}}
```

Comment:
should we go ahead and just provide the ability to login with any provider/email-password in one page? like adding the github login here and moving it out of the sidenav